sale_order_price_recalculation/README.rst

```diff
@@ -0,0 +1,63 @@
+===================================================
+Recalculation of sales order lines prices on demand
+===================================================
+
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
```

Comment:
We can see that the page is up to date on apps.odoo.com as there are empty <a> balises for the logo